.github/workflows/conformance-datapath.yaml

```diff
@@ -226,7 +226,8 @@ jobs:
             --config monitor-aggregation=none \
             --nodes-without-cilium=kind-worker3 \
             --helm-set-string=kubeProxyReplacement=${{ matrix.kpr }} \
-            --helm-set=bpf.masquerade=false"
+            --helm-set=bpf.hostLegacyRouting=true \
+            --helm-set=egressGateway.enabled=true"
```

Comment:
I think it makes sense to enable the EGW feature only if the `kubeProxyReplacement=strict` above, as the former depends on the latter.

Thinking out loud, maybe we could have a separate section in the configuration matrix for the EGW (e.g,. `egress-gateway: true`) to enable it?

Reply:
Yeah, my bad.